dev/io.openliberty.io.grpc.1.0.jakarta/bnd.bnd

```diff
@@ -0,0 +1,38 @@
+#*******************************************************************************
+# Copyright (c) 2020, 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR.
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+grpcVersion=1.38.1
+
+Bundle-SymbolicName: io.openliberty.io.grpc.1.0.javaee; singleton:=true
```

Comment:
Why include `1.0` in the name of the bundle?

Reply:
this is an API bundle - we followed the example of `io.openliberty.example.1.0` per https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines

Reply:
this is a _stable_ API bundle - based on https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines we thought the 1.0 made sense